systemtap/systemtap.install

```diff
@@ -1,8 +1,4 @@
 post_install() {
-  getent group stapusr >/dev/null || groupadd -g 156 -r stapusr 2>/dev/null || groupadd -r stapusr
-  getent group stapsys >/dev/null || groupadd -g 157 -r stapsys 2>/dev/null || groupadd -r stapsys
-  getent group stapdev >/dev/null || groupadd -g 158 -r stapdev 2>/dev/null || groupadd -r stapdev
-
   getent group stapusr >/dev/null && chgrp stapusr /usr/bin/stapbpf
```

Comment:
`chgrp stapusr` should be `chgrp 156` as the `post_install()` function is run before the `.sysusers` file is processed.

Reply:
Furthermore, chgrp commands are better to be in `package()` of PKGBUILD.

Reply:
If `chgrp` commands are in `post_install()`, `pacman -Qkk systemtap` will report GID mismatch for these files. That's a minor issue, anyway.

Reply:
OK, moved.